tutorials/using-bucket-website-with-mkdocs/index.mdx

```diff
@@ -0,0 +1,380 @@
+          ```
+    #### Adding admonition icons
+
+        Copy and paste the following code template to use Octicons admonition icons. You can also use the [FontAwesome admonition icons](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admonitions#admonition-icons-fontawesome).
+
+        ```
+          icon:
+            admonition:
+              note: octicons/tag-16
+              abstract: octicons/checklist-16
+              info: octicons/info-16
+              tip: octicons/squirrel-16
+              success: octicons/check-16
+              question: octicons/question-16
+              warning: octicons/alert-16
+              failure: octicons/x-circle-16
+              danger: octicons/zap-16
+              bug: octicons/bug-16
+              example: octicons/beaker-16
+              quote: octicons/quote-16
+        ```
+
+    #### Using admonitions
+
+        The example below shows you the syntax you need to use in your files for admonitions. You can replace `info` with any of the admonitions you have added in the previous section (`abstract`, `note`, `tip`, `success`, `question`, `warning`, `failure`, `danger`, `bug`, `example`, and `quote`).
+
+        ```
+        !!! info
+
+            I am an information admonition. You can use me to give additional information to your user.
+        ```
+
+        You can add a custom title inside the admonition box like in the example below.
+
+        ```
+        !!! info "Information about the admonition box"
+
+            I am an information admonition. You can use me to give additional information to your user.
+        ```
+
+        If you want your admonition box to be collapsible, you can use the code below. The user will only see the box without its content.
+
+        ```
+        ??? info
+
+            I am an information admonition. You can use me to give additional information to your user.
+        ```
+
+        Find out more about admonitions on the [MkDocs website](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admonitions#configuration).
+
+    ### Additional configuration
+
+        You can add the following additional configuration options in your `mkdocs.yml` file. The `features` component allows you to use various features of the Material theme.
+          ```
+          features:
+            - content.code.copy
+            # Adds a copy button to code blocks.
+            - navigation.sections
+            # Enables section-based navigation.
+            - navigation.path
+            # Shows the navigation path (breadcrumbs).
+            - navigation.top
+            # Adds a "back to top" button.
+            - search.suggest
+            # Enables search suggestions.
+          ```
+        Add the `plugins` component to define the MkDocs plugins you want to enable on your website. The `search` plugin allows you to add the search functionality and the `glightbox` plugin allows you to add images and videos.
```

Comment:
Maybe explain how to add images since we give users the configuration to add images